incubator/jaeger/README.md

```diff
@@ -313,6 +313,7 @@ The following table lists the configurable parameters of the Jaeger chart and th
 | `schema.pullPolicy`                      | Schema image pullPolicy             |  `IfNotPresent`                          |
 | `schema.activeDeadlineSeconds`           | Deadline in seconds for cassandra schema creation job to complete |  `120`                            |
 | `schema.traceTtl`                     | Time to live for trace data in seconds      |  `nil`                                |
+| `schema.keyspace`                     | Set implicit keyspace name      |  `nil`                                |
```

Comment:
Thanks for the PR! Don't you mean to say "explicit" here?

Reply:
Yes i did, it has been changed